CI/travis/before_install_linux

```diff
@@ -20,7 +20,7 @@ handle_ubuntu_docker() {
 
 handle_default() {
 	sudo apt-get -qq update
-	sudo apt-get install -y cmake doxygen libaio-dev libavahi-client-dev libavahi-common-dev libusb-1.0-0-dev libxml2-dev rpm tar bzip2 gzip flex bison git
+	sudo apt-get install -y cmake doxygen graphviz libaio-dev libavahi-client-dev libavahi-common-dev libusb-1.0-0-dev libxml2-dev rpm tar bzip2 gzip flex bison git
```

Comment:
do we also need to do the same for CentOS here https://github.com/analogdevicesinc/libiio/blob/master/CI/travis/inside_centos_docker.sh#L8 ?

Reply:
No - the doxygen html isn't actually deployed anywhere (yet), and it just takes extra time.

Reply:
ack